client/config/webpackDevServer.config.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['use strict' is unnecessary inside of modules.](https://app.codacy.com/manual/oceanprotocol/commons/pullRequest?prid=5420705)